default.yaml

```diff
@@ -124,6 +124,19 @@ rateLimits:
 
 accounts:
   initialStatus: active
+  information:
```

Comment:
this is just test data  to be able to see the changes in the graphql schema file

Reply:
I would call it `custom` or `customFields`

Reply:
updated https://github.com/GaloyMoney/galoy/pull/1465/commits/c536cd781df7aba2f2611bb8675e4ac38188f347